guide/payments/contacts.md

```diff
@@ -0,0 +1,229 @@
+      alt: Payment request screen
+      caption: An invoice screen with minimal information.
+    - file: receive-qr-with-contact
+      alt: Payment request screen with a contact assigned
+      caption: The same invoice with a contact assigned.
+imagesActivity:
+    - file: activity
+      alt: Activity screen showing transactions with minimal information
+      caption: By default, payments contain little to no contextual information.
+    - file: activity-annotated
+      alt: Activity screen showing transactions with assigned contacts
+      caption: Assigning contacts to payments makes them much easier to understand and work with.
+    - file: activity-transaction
+      alt: Transaction details
+      caption: Transaction detail screens should have convenient options to assign a contact.
+    - file: sending-pick-contact
+      alt: Screen with a list of contacts to choose from
+      caption: The user picks a contact in a slide-up modal.
+    - file: activity-transaction-annotated
+      alt: Transaction details with assigned contact
+      caption: A payment with an assigned contact.
+
+---
+
+{% include picture.html
+    image = "/assets/images/guide/payments/contacts/contacts-header.jpg"
+    retina = "/assets/images/guide/payments/contacts/contacts-header@2x.jpg"
+    mobile = "/assets/images/guide/payments/contacts/contacts-header-mobile.jpg"
+    mobileRetina = "/assets/images/guide/payments/contacts/contacts-header-mobile@2x.jpg"
+    alt-text = "Illustration of large contact cards"
+    width = 1600
+    height = 400
+    layout = "full-width"
+%}
+
+
+# Contacts
+{:.no_toc}
+
+<!--
+
+
+-->
+
+Whether we’re sending emails, physical mail, or following someone on social media, we primarily think in terms of names and faces, and not the respective address or user ID.
+
+Invoices, node IDs and other transaction endpoints in bitcoin and Lightning are highly unintuitive. Abstracting them via a contact list can create a much smoother user experience. There are many [payment request formats]({{ '/guide/payments/request/payment-request-formats/' | relative_url }}), each with unique properties and varying levels of maturity and adoption, requiring unique design solutions. This page uses the more approachable term "address", along with various UI techniques, to abstract these complexities for users.
+
+Let's go over common user interactions around contacts. This will illustrate how such a feature could work, and helps explain the underlying design problems and decisions.
+
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
+{:toc}
+</div>
+
+---
+
+### Adding a contact
+
+The most basic interaction is that a user manually adds a contact by entering their name and an address. This is likely not the most common user flow, as most contacts will be added to incoming payment requests, as illustrated further below. But wallets should generally support manual options, as users may not be able to avoid them.
```

Comment:
```suggestion
The most basic interaction is that a user manually adds a contact by entering their name and an address. This is likely not the most common user flow, as most contacts will be created with incoming payment requests, as illustrated further below. But wallets should generally support manual options, as users may not be able to avoid them.
```